db/migrate/20240603131021_create_waste_exemptions_engine_orders.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class CreateWasteExemptionsEngineOrders < ActiveRecord::Migration[7.1]
+  def change
+    create_table :orders, &:timestamps
```

Comment:
I'd expect it to have more attributes, e.g. Order number and some user details (at least user email), so it can be linked with particular person placing an order

Reply:
or link to account (if we have one)

Reply:
See my response to your first comment. I agree with you but I think not on this ticket. Happy to discuss.